test/e2e/bootstrap_resources.py

```diff
@@ -16,19 +16,17 @@
 """
 
 from dataclasses import dataclass
-from acktest.resources import read_bootstrap_config
+from acktest.bootstrapping import Resources
 from e2e import bootstrap_directory
 
 @dataclass
-class TestBootstrapResources:
+class TestBootstrapResources(Resources):
```

Comment:
Please rename this to `BootstrapResources`. I overlooked this when I initially created it, but starting any class with `Test`, pytest tries to run it as a test.